app/app/dashboard/page.tsx

```diff
@@ -1,4 +1,102 @@
+"use client";
+import { useAuth0 } from "@auth0/auth0-react";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import { formatGreetingTime } from "@/utils/formatters";
+import { FaMap, FaChartPie } from "react-icons/fa6";
+import { formatFirstNameFromEmail } from "@/utils/auth";
+import DashboardLinkCard, {
+  DashboardLinkCardProps,
+} from "@/components/DashboardLinkCard";
+
+const VZV_ENDPOINT = process.env.NEXT_PUBLIC_VZV_ENDPOINT || "";
+
+const dashboardLinks: DashboardLinkCardProps[] = [
+  {
+    title: "Arterial Management Division Overview",
+    description: "Top location overview, by collision types and modes",
+    href: "https://app.powerbigov.us/links/GACOsce5fi?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare",
+    icon: FaChartPie,
+  },
+  {
+    title: "Vision Zero Viewer",
+    description:
+      "Our public-facing dashboard with official Vision Zero statistics",
+    href: "https://visionzero.austin.gov/viewer/",
+    icon: FaMap,
+  },
+  {
+    title: "Emerging Hotspots and Bond Locations",
+    description:
+      "Track crash impact of Vision Zero Bond Projects and changing crash trends",
+    href: "https://app.powerbigov.us/links/RmMrnaSMLp?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare",
+    icon: FaChartPie,
+  },
+  {
+    title: "Crash Data by Location",
+    description: "Based on Vision Zero location polygons",
+    href: "https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=981b687bb68d465cab737792bbf56198",
+    icon: FaMap,
+  },
+  {
+    title: "High Injury Roadways",
+    description: "Each High Injury Roadway by Polygon with various statistics",
+    href: "https://app.powerbigov.us/links/pdguGuhSGE?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare",
+    icon: FaChartPie,
+  },
+  {
+    title: "Access Management Crashes",
+    description: "Summary of crashes for individual locations",
+    href: "https://austin.maps.arcgis.com/apps/instant/interactivelegend/index.html?appid=ea84226e6e5a4ecf998082b73b8c6cca",
+    icon: FaMap,
+  },
+];
+
 export default function Dashboard() {
-    return <h1>This is the dashboard page</h1>;
-  }
-  
\ No newline at end of file
+  const { user } = useAuth0();
+  const firstName = formatFirstNameFromEmail(user?.email || "");
+
+  return (
+    <>
+      <Row className="mb-3">
+        <Col md={9}>
+          <h6 className="display-6">
+            <span>{`Good ${formatGreetingTime(new Date())}, `}</span>
+            <span className="text-capitalize">{firstName}</span>!
+          </h6>
+          <p>
+            Welcome to the Vision Zero Editor. The below metrics reflect
+            our&nbsp;
+            <a href={VZV_ENDPOINT} target="_blank" rel="noreferrer">
+              public crash data
+            </a>
+            , which exclude crashes which occurred within the last 14 days.
```

Comment:
I like Rose's version better 
>"Welcome to the Vision Zero Editor. The below metrics reflect our [public crash data](http://localhost:3000/viewer), which excludes crashes that occurred within the last 14 days."
With the copy you have I thought excludes would subject/verb agree better than exclude, but style guides say both work. its Friday and im tired,

Reply:
I am apparently on a which hunt (ba dum ch ...sorry 😞 )